src/components/Card.tsx

```diff
@@ -1,19 +1,17 @@
-import { useState } from 'react';
+import { useState, useMemo } from 'react';
 import { History, MurojaahType } from '@/api/module/murojaah/entity';
 import { formatDatetime } from '@/util/datetime';
 import { clsx } from 'clsx';
 
 export const Card = (item: History): JSX.Element => {
-  if (!Object.values(MurojaahType).includes(item.murojaahType)) return <></>;
-
   const cardClassnames: Record<string, string> = {
```

Comment:
we can move this out of Card component, so that it doesn't have to be a dependency of useMemo below

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/9/commits/6b48662e81200eacafdfe4be36e43276682066fc, thank you